SQL/Archive/17.0/2016-08-19-IssueTracker.sql

```diff
@@ -0,0 +1,106 @@
+-- Add issues tab to the Loris Menu
+INSERT INTO `LorisMenu` (`ID`, `Parent`, `Label`, `Link`, `Visible`, `OrderNumber`)
+VALUES (61, 5, 'Issue Tracker', 'issue_tracker/', NULL, 8);
```

Comment:
* 61,5 --> use subselect to look up rather than hardcode these values, particularly for existing projects
* Visible NULL -- this should probably be true by default.  Can't think of a project that wouldn't want to use this module.